### ISSUE TITLE
Hello, can this method be integrated into FairMOT?

### ISSUE THREAD

**Issue by ydhexyk** [ymzis69/HybirdSORT#7]

**Comment by ymzis69** [comment]
Of course, you can take a look at our implementation in ByteTrack for reference. It's not difficult to implement.

**Comment by ydhexyk** [comment]
Thank you for your reply.

**Comment by ydhexyk** [comment]
FairMOT is not a TBD paradigm but belongs to JDE. Would this affect the application of this method?Will this lead to an increase in difficulty?

**Comment by ymzis69** [comment]
We applied TCM to methods like DeepSORT, SORT, etc., and observed a certain degree of improvement. This demonstrates the generality of our approach. FairMOT is similar to DeepSORT, should also benefit from TCM.

**Comment by HanGuangXin** [comment]
> FairMOT is not a TBD paradigm but belongs to JDE. Would this affect the application of this method?Will this lead to an increase in difficulty?

@ydhexyk To be honest, TBD paradigm includes both SDE and JDE paradigm when ReID module are employed. Hybrid-SORT could be readily applied to all SORT-like method, because Hybrid-SORT only introduces the new cost matrix which is generated using newly incorporated weak cues (i.e., TCM) or modified from the existing cost matrix (i.e., HMIoU and ROCM).

**Comment by HanGuangXin** [comment]
Closing the issue. Please feel free to reopen if there are further questions.
